rosetta/retriever/retriever_test.go

```diff
@@ -361,45 +395,45 @@ func TestRetriever_Block(t *testing.T) {
 		generator := mocks.BaselineGenerator(t)
 		generator.TokensDepositedFunc = func(symbol string) (string, error) {
 			assert.Equal(t, symbol, dps.FlowSymbol)
-			return string(mocks.GenericEventType(0)), nil
+			return string(withdrawalType), nil
 		}
 		generator.TokensWithdrawnFunc = func(symbol string) (string, error) {
 			assert.Equal(t, symbol, dps.FlowSymbol)
-			return string(mocks.GenericEventType(1)), nil
+			return string(depositType), nil
 		}
 
 		convert := mocks.BaselineConverter(t)
 		convert.EventToOperationFunc = func(event flow.Event) (*object.Operation, error) {
-			assert.Contains(t, mocks.GenericEvents(4), event)
-
 			var op object.Operation
 			switch event.Type {
-			case mocks.GenericEventType(0):
+			case withdrawalType:
```

Comment:
You already do this most of the time, but then not completely consistently. I definitely find it much more readable after the change, and would welcome that to be consistently applied to all tests.

Reply:
I'll need to go over them again then, I thought I did it consistently everywhere, but I guess not :p